src/ui/util/columns/templates/Name.tsx

```diff
@@ -0,0 +1,14 @@
+import type { TemplateProps } from "../getCols";
+import { PlayerNameLabels } from "../../../components";
+
+export default ({ p, c, vars }: TemplateProps) => (
+	<PlayerNameLabels
```

Comment:
What if additional props need to be passed to PlayerNameLabels? What if they only need to be conditionally passed? For example https://github.com/zengm-games/zengm/blob/8a59d0928d770dffe0a14d97d7063b4a0fe05a74/src/ui/views/PlayerStats.tsx#L139

Reply:
In the `MetaCol` type there is a property called `options` and there could be an option for season.
However because it is likely more than 1 column needs to know what season is selected, I've added it to `TableConfig.vars` property

Reply:
It all gets very messy and complicated... because in some places, "career" is a valid season, but in other places it's not. Ideally there would be some TypeScript-enforced way to say "the rows of data passed to the table need these properties" but even that is hard to think of what exactly it means, given what I mentioned above about some variables meaning different things in different tables. This is going to be so hard to get right without bugs/regressions. Especially since `playersPlus` returns `any` currently.

It's possible we'll need some way to pass accessor functions to template functions, like a way to say "by default skills is in p.ratings.skills, or supply a callback function that takes p and returns skills".